gitlab/resource_gitlab_managed_license.go

```diff
@@ -0,0 +1,138 @@
+package gitlab
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
```

Comment:
Change to SDK v2, like:

```suggestion
	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
```